python/chronos/src/bigdl/chronos/data/repo_dataset.py

```diff
@@ -92,3 +96,36 @@ def get_public_dataset(name, path='~/.chronos/dataset', redownload=False, **kwar
                           "Only network_traffic, AIOps, fsi, nyc_taxi, uci_electricity"
                           " uci_electricity_wide"
                           f"are supported in Chronos built-in dataset, while get {name}.")
+
+
+def gen_synthetic_data(len=10000, **kwargs):
+    """
+    Generate dataset according to sine function y=10*sin(PI*x/1000).
```

Comment:
try give more flexibility to the synthetic data, e.g. 10 and 1000 could be 2 parameters, a Gaussian noise can be added to the data as well (with another parameter indicates the intensity)